.buildkite/Dockerfile

```diff
@@ -1,34 +1,50 @@
-FROM docker.io/python@sha256:832385dfbafed97b56379fa2665f858d9178f8fea4bc6efc8d3282804d68a6b4
+FROM docker.io/buildpack-deps:22.04-curl
 
 ENV PYTHONUNBUFFERED=1 \
 	PYTHONDONTWRITEBYTECODE=1 \
 	PIP_NO_CACHE_DIR=off \
 	PIP_DISABLE_PIP_VERSION_CHECK=on \
 	PIP_DEFAULT_TIMEOUT=100 \
-	PATH="$PATH:/root/.local/bin/"
+	PATH="$PATH:/root/.local/bin/:/.pyenv/bin/" \
+	PYENV_ROOT="/.pyenv"
 
 RUN apt-get -y update && \
-    apt-get -y install curl build-essential gcc git swig && \
+    apt-get -y install curl build-essential gcc git swig libffi-dev libncurses5-dev zlib1g zlib1g-dev libssl-dev libsqlite3-dev liblzma-dev libreadline-dev libbz2-dev && \
     apt-get -y clean
 
+RUN curl https://pyenv.run | bash && \
+	eval "$(pyenv init -)" && \
+	pyenv install 3.11 && \
+	pyenv global 3.11 && \
+	pyenv install 3.10 && \
+	pyenv install 3.9 && \
+	pyenv install 3.8 && \
+	pyenv rehash && \
+	python --version && \
+	pyenv exec pip --version
+
 RUN mkdir -p /gcloud/ && \
 	curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-408.0.1-linux-x86_64.tar.gz && \
 	tar -xf google-cloud-cli-408.0.1-linux-x86_64.tar.gz -C /gcloud/ && \
 	/gcloud/google-cloud-sdk/install.sh --quiet --usage-reporting false && \
 	rm google-cloud-cli-408.0.1-linux-x86_64.tar.gz
 
-RUN pip install \
-	--no-cache-dir \
-	pipx && \
+RUN eval "$(pyenv init --path)" && \
+	eval "$(pyenv init -)" && \
```

Comment:
Is `pyenv init -` meant to be run twice?

Reply:
It has to be run with `-` and `--path` apparently.